contributors/index.js

```diff
@@ -1422,6 +1422,14 @@ const list = [{
 		github_username: 'ramanujamgond',
 		twitter_username: 'string1301',
 	},
+	      	{
```

Comment:
It has Spacing issues. Please check for these types linting issues by running `npm run lint`.and `npm run lint:fix` to fix it before pushing it.